Tactical/LOS.cpp

```diff
@@ -5304,15 +5296,7 @@ INT8 FireBulletGivenTarget( SOLDIERTYPE * pFirer, FLOAT dEndX, FLOAT dEndY, FLOA
 	dDeltaX = dEndX - dStartX;
 	dDeltaY = dEndY - dStartY;
 	dDeltaZ = dEndZ - dStartZ;
-#if 0//dnl ch60 311009 this isn't correct, e.g. if you try head shot from prone to standing target at 1 tile, Distance3D will calculate 7,8 tile distance, this means that you will always hit target regard of CTH calculation, correct usage will be Distance3D(dDeltaX, dDeltaY, CONVERT_HEIGHTUNITS_TO_DISTANCE(dDeltaZ)) but we need here 2D distance as was in original v1.12

```

Comment:
preserve comment about Distance2D call, in case we ever want to fix this part to use Distance3D?